### ISSUE TITLE
[BUG] No signing key warning spam in development

### ISSUE THREAD

**Issue by ianschmitz** [inngest/inngest-js#356]
**Describe the bug**
After the change in https://github.com/inngest/inngest-js/pull/125, our console now gets a lot of warning messages of the following:

```
No signing key provided to validate signature. Find your dev keys at https://app.inngest.com/test/secrets
```

Reading the PR, i see this looks to be an intentional change. However it's quite spammy in development and covers up other possible legitimate error messages i might be missing from Next.

**To Reproduce**

1. Use the SDK as intended in a local development environment

**Expected behavior**
Not to spam my console during local development when using the SDK as intended.

**Code snippets / Logs / Screenshots**

![image](https://github.com/inngest/inngest-js/assets/6355370/6b03c5ed-7360-4e8a-9b9d-8e2361edebdd)

**System info (please complete the following information):**

- OS: macOS
- npm package Version 3.1.1
- Framework Next.js
- Platform N/A

**Additional context**

**Comment by transitive-bullshit** [comment]
This is really confusing for a new dev coming in.

**Comment by jpwilliams** [comment]
This fixed in [`3.4.4`](https://github.com/inngest/inngest-js/pull/390).
